src/client/templates/export.html

```diff
@@ -0,0 +1,15 @@
+<template name="exportRoom">
```

Comment:
I don't know how, but we need to figure out how to not have the normal page styling (header_footer, we want this as a plain-text only page. It's supposed to be an easily copy-pasta, but when I did and put into something like a drive folder, it looks like this http://i.imgur.com/MZaCPnU.png
I honestly would even say drop the HTML altogether and just use plain text